src/api/app/controllers/webui/requests/submissions_controller.rb

```diff
@@ -54,6 +54,8 @@ def supersede_requests
                                  superseded_by: @bs_request.number)
         rescue APIError => e
           supersede_errors << e.message.to_s
+        rescue ActiveRecord::RecordNotFound
+          supersede_errors << "Couldn't find request with id '#{request_number}'"
```

Comment:
To have a nicer message for users whenever something goes wrong as they submit a BsRequest. This keeps the same behavior as before.